rclpy/rclpy/node.py

```diff
@@ -1019,11 +1048,11 @@ def set_parameters_callback(
         """
         Register a set parameters callback.
 
-        Calling this function with override any previously registered callback.
+        Calling this function will add a callback in self._parameter_callbacks list.
 
         :param callback: The function that is called whenever parameters are set for the node.
         """
-        self._parameters_callback = callback
+        self._parameters_callbacks = [callback]
```

Comment:
@dirk-thomas there's a ticket, ros2/rclcpp#791, to deprecate this call for C++. Should an equivalent ticket be added for Python?